### PR TITLE
second change another

### PR DESCRIPTION
**Pull request by Andrey-1313**
second change another

### CODE REVIEW

**Review by testuser131313 — COMMENTED** [review]
add comments

**Comment by Andrey-1313** [comment]
> 

added comments

File: another.py [hunk 1]
```diff
@@ -1,2 +1,3 @@
 for i in range(2, 10):
   print (f'format {i}')
```
Comment (testuser131313) [inline another.py:2]: change to 1

File: another.py [hunk 2]
```diff
@@ -1,2 +1,3 @@
 for i in range(2, 10):
```
Comment (testuser131313) [inline another.py:1]: change to 3

**Pull request merged**